feat(external): add mrvi (#2756)
scverse/scvi-tools: CHANGELOG.md

    {pr}`2692`.
- Add support for custom dataloaders in {class}`scvi.model.base.VAEMixin` methods by specifying
    the `dataloader` argument {pr}`2748`.

> - Add {class}`scvi.external.MRVI` for hierarchical modeling of single-cell RNA-seq data {pr}`2756`.

Reviewer: ```suggestion
- Add {class}`scvi.external.MRVI` for modeling sample-level heterogeneity in single-cell RNA-seq data {pr}`2756`.
```
